### PR TITLE
Chore/pr review edits

### PR DESCRIPTION
**Pull request by kieha**
**What does this PR do?**

PR review edits

**Task to be completed?**

Changes incorporated after checkpoint review with trainer

### CODE REVIEW

File: server/controllers/users.js [hunk 1]
```diff
@@ -18,18 +18,33 @@ module.exports = {
...
+            error = err.errors['name.lastname'].message.replace(/name\./gi, '');
+          }
+          res.status(400).send({ error: `Error creating user: ${error}` });
         }
       } else {
         const userData = {
           _id: user._id,
           email: user.email,
+          role: user.role,
         };
```
Comment (andela-kakpobome) [inline server/controllers/users.js:48]: ## Nitpick

Indentation is off here

Comment (kieha) [reply server/controllers/users.js:48]: @andela-kakpobome which line?

Comment (andela-kakpobome) [reply server/controllers/users.js:48]: @andela-skieha line 48

Comment (kieha) [reply server/controllers/users.js:48]: @andela-kakpobome the closing curly brace on line 48 is closing the userData object, so it's not really off

Comment (andela-kakpobome) [reply server/controllers/users.js:48]: Missed that, thank you for pointing out.

**Pull request merged**